CONTRIBUTING.md

```diff
@@ -260,16 +211,4 @@ This section will change much more frequently compared to the other sections.
-
-[Graphile worker](https://github.com/graphile/worker) is a job queue that runs on PostgreSQL + Node.js. The engine has one Graphile worker per database. Graphile worker setup and job definition is defined in the [scheduler.ts](https://github.com/iasql/iasql-engine/blob/main/src/services/scheduler.ts). At startup, the Node.js engine looks at its [metadata repository](https://github.com/iasql/iasql-engine/blob/main/src/entity/index.ts) which stores information about all the IaSQL databases the engine is managing so it can spin up a Graphile worker per existing IaSQL database. New databases created via the `/connect` route get a new metadata repo entry and a new Graphile worker within the workers' process.
-
-<img src="https://user-images.githubusercontent.com/5357435/187452721-2b9eab4a-f007-406b-91d8-5467620d8058.png" width="500">
-
-Graphile workers are contacted via HTTP calls to the Express.js server defined in [`scheduler.ts`](src/services/scheduler.ts) so thatthe IaSQL functions in SQL can call into the actual logic of operations that live in the Node.js engine. The PG SQL functions poll until the Node.js code is done via the [`iasql_operation`](https://github.com/iasql/iasql-engine/blob/main/src/modules/0.0.18/iasql_functions/entity/index.ts) table which is used to keep the output, the state, and the finish time for every IaSQL operation (`apply`, `sync`, `install`, etc.) in that database. The `until_iasql_operation` PG SQL function is ultimately called by every IaSQL function and is what kicks off the polling. This function invokes the Node.js code via the database's Graphile worker by using `graphile_worker.add_job(...)` and then creates a while loop watching for the `end_time` column in `iasql_operation` table to wait for the result of the Node.js code.
-
-![](https://user-images.githubusercontent.com/5357435/185755422-84c7c9d2-f5e4-4c6f-8a16-ef8d7601644f.png)
-
-However, PostgreSQL does not support autonomous transactions and all PG functions run within a transaction. An autonomous transaction is an independent transaction that runs within a parent transaction that must be committed or rolled back before the parent transaction is finished. This means that PostgreSQL does not allow, within a single transaction or function, to poll for the result of another process. We simulate autonomous transactions to allow polling by using a whole new connection via dblink as described [here](https://aws.amazon.com/blogs/database/migrating-oracle-autonomous-transactions-to-postgresql/) which is used to call `graphile_worker.add_job(...)`. The engine keeps a [dblink server](https://github.com/iasql/iasql-engine/blob/main/src/services/scheduler.ts#L29) per database with the extra connection that can be re-used for polling. This is how we are able to keep a synchronous UX with a vanilla Postgres instance without any extensions for the time being.
+<!-- TODO: Revive Postgres function explanation for the new HTTP-based RPC approach -->
```

Comment:
Also created a new issue for this https://github.com/iasql/iasql-engine/issues/1712